conduct/code-of-conduct.md

```diff
@@ -38,13 +38,6 @@ harassment purposes.
 except as necessary to protect vulnerable people from intentional abuse.
 * Publication of non-harassing private communication.  
 
-`<EVENT>` prioritizes marginalized people's safety over privileged people's comfort. We will not act on complaints regarding:
-
-* 'Reverse' -isms, including 'reverse racism,' 'reverse sexism,' and 'cisphobia'.
-* Reasonable communication of boundaries, such as "leave me alone," "go away," or "I'm not discussing this with you."
-* Communicating in a 'tone' you don't find congenial.
-* Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions.
-
 ### Harassment in online channels
 
 Participants at `<EVENT>` may also participate in online chats, discussion forums, and mailing lists (including platforms such as Slack and Twitter). Please use these guidelines where engaging with particpants:
```

Comment:
cc @karawoo @gdequeiroz 

### Suggestion for altering this "Harassment in online channels" section

change first paragraph to

> Participants at `<EVENT>` may also participate online (see below). Please use these guidelines where engaging with particpants:

Then add below the two bullets: 

This CoC applies to the all `<EVENT>` online spaces, which includes, but is not limited to:

* Twitter
* Slack
* Facebook
* WhatsApp
* Gitter

This CoC does not exclusively apply to online channels during the conference. For example, if someone is harassed in a conference related online channel leading up to the event while discussing ideas for the event, it would be treated as a CoC violation.